data/companies.json

```diff
@@ -336,6 +336,22 @@
         "youtube": "https://www.youtube.com/c/Eminenceways"
       }
     },
+    {
+      "name": "e-Zone International",
+      "establishedAt": 1049155200,
+      "location": "Mahalaxmisthan, Lalitpur, Nepal",
+      "website": "https://www.ezone.com.np/",
+      "email": "info@ezone.com.np",
+      "phone": "+977-9802026676",
```

Comment:
Does this place have a landline number? If so, would you mind replacing it with that?

Reply:
@Yush-Pokharel

Reply:
Updating it right away.